deemix: add page (#4381)
tldr-pages/tldr: pages/common/deemix.md

# deemix

> A barebone deezer downloader library built from the ashes of Deezloader Remix.

> > It can be used as a stand alone CLI app or implemented in an UI using the API.

Reviewer: standalone

Author: fixed!

> More Information: <https://deemix.app>.

- Download a track or playlist:

> `deemix {{https://www.deezer.com/us/track/108441824}}`

Reviewer: I'd prefer to give a dummy track that does not resolve to a page.

- Download track / playlist at a specific bitrate:

> `deemix --bitrate {{FLAC / MP3}} {{url}}`

Author: I have tried using multiple variations of the bitrate but it always ends up ignoring them but using file types always returns the correct bitrates. i.e 320kbps for MP3 and 1411kbps for FLAC.
It might be a bug, it might be the way it was coded but since there's no proper documentation I can't say for sure.

> `deemix --bitrate {{FLAC / MP3}} {{url}}`

Reviewer 2: Ah, I see. So the filetypes are actually presets. What about this then? It would make it more consistent with other pages.
```suggestion
`deemix --bitrate {{FLAC|MP3}} {{url}}`
```

- Download to a specific path:

`deemix -bitrate {{bitrate}} --path {{path}} {{url}}`

> - Create a portable deemix config (will be created in the present working directory):

Reviewer 3: Is this example description more clear?
```suggestion
- Create a portable deemix config in the current directory:
```
